en/components/splitter.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Angular Splitter | Split Panes |  Ignite UI for Angular | Infragistics
+_description: Use the Angular Splitter component to create a simple split layout splitting the view horizontally or vertically into multiple collapsible split panes.
+_keywords: angular splitter, igniteui for angular, infragistics
+---
+
+## Splitter
+
+The Ignite UI for Angular Splitter component provides a framework for a simple split layout splitting the view horizontally or vertically into multiple smaller areas called split panes.
```

Comment:
"simple split layout splitting ... into ... split panes"
Seems a bit redundant.

Can omit some of the 'split's:
"simple layout splitting ... into ... panes" 

Also we can mention that the panes can be expanded/collapsed as by default we also show buttons for those operations and they will be visible in the demo below.

Reply:
I removed some of the "split"-s from first sentence, but added "split panes" in the second one. This is a keyword which is good to be included in the first paragraph.